scripts/refresh-on-new-extension.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+REPO="${LOCALDEV_REPO:-/repo}"
+
+inotifywait -q -m -r -e CLOSE_WRITE $1 | while read DIRECTORY EVENT FILE; do
```

Comment:
💪  is this only on "new" files?  What is the file is "updated"?

Reply:
In the case of a file being "updated" tilt will already refresh, so this would cause double-refresh.  
Could we do this instead?
```
inotifywait -q -m -r -e CREATE $1 | while read DIRECTORY EVENT FILE; do
```

Reply:
Updating client-extension.yaml does not trigger via inotify.

Reply:
.. due to how it's configured.